builtin/client/chatcommands.lua

```diff
@@ -40,8 +40,14 @@ end)
 core.register_chatcommand("list_players", {
 	description = core.gettext("List online players"),
 	func = function(param)
-		local players = table.concat(core.get_player_names(), ", ")
-		core.display_chat_message(core.gettext("Online players: ") .. players)
+	    local player_names = core.get_player_names()
+	    if player_names == nil then
+	        core.display_chat_message(core.gettext("This command is restricted by server."))
```

Comment:
'disabled by server' is better English, 'restricted' doesn't really specify if it's disabled or not.